community/kernel/src/main/java/org/neo4j/kernel/builtinprocs/BuiltInProcedures.java

```diff
@@ -162,8 +164,11 @@
                         type = IndexType.NODE_LABEL_PROPERTY.typeName();
                     }
 
-                    result.add( new IndexResult( "INDEX ON " + index.schema().userDescription( tokens ),
-                            operations.indexGetState( index ).toString(), type ) );
+                    String label = tokens.labelGetName( index.schema().getLabelId() );
+                    List<String> propertyNames = propertyNames( tokens, index );
+                    result.add( new IndexResult( "INDEX ON " + index.schema().userDescription( tokens ), label, propertyNames,
```

Comment:
how is current index description message will gonna look like?

Reply:
it will look like
```
{
  "version": "1.0",
  "key": "lucene"
}
```